bibtex-completion.el

```diff
@@ -352,6 +360,11 @@ more types can slow down resolution for large biblioraphies.")
   "Stores `bibtex-completion-display-formats' together with the \"used width\" of each format string.
 This is set internally.")
 
+
+(defvar bibtex-completion-display-formats-suffix-internal nil
```

Comment:
```suggestion
(defvar bibtex-completion--display-formats-suffix-internal nil
```
I was wanting to make these symbol name private using the `--` convention, but have not since it's not currently used anywhere else in the code. Would be nice to add this at some point.